org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,228 @@
+                                        "url-template": "https://www.freedesktop.org/software/uchardet/releases/uchardet-$version.tar.xz"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "squashfs-tools",
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "XZ_SUPPORT=1 LZO_SUPPORT=0 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "git",
+                                   "url": "https://github.com/plougher/squashfs-tools.git",
+                                   "tag": "4.6.1",
+                                   "commit": "d8cb82d9840330f9344ec37b992595b5d7b44184",
+                                   "x-checker-data": {
+                                        "type": "git",
+                                        "tag-pattern": "^([\\d.]+)$"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "musicpod",
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "unsquashfs -quiet -no-progress *.snap",
+                              "cp -r squashfs-root/bin /app",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.desktop /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "desktop-file-edit --set-key=Icon --set-value='org.feichtmeier.Musicpod' /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.png /app/share/icons/hicolor/256x256/apps/org.feichtmeier.Musicpod.png",
+                              "install -Dm644 org.feichtmeier.Musicpod.appdata.xml /app/share/metainfo/org.feichtmeier.Musicpod.appdata.xml"
+                         ],
+                         "cleanup": [
+                              "/include",
+                              "/lib/pkgconfig",
+                              "/share/man"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "file",
+                                   "path": "org.feichtmeier.Musicpod.appdata.xml",
+                                   "sha256": "aec5891c54a81ab64799b1e9efa05899d2ae16503313168152dae80f25523ce2",
+                                   "url": "https://raw.githubusercontent.com/ubuntu-flutter-community/musicpod/main/flatpak/org.feichtmeier.Musicpod.appdata.xml"
```

Comment:
you should NEVER reference a branch. Always a specific commit.

Reply:
I think the thought process behind that was it wouldn't have to keep updating.

What needs changed the sha256 to a specific commit?

@Feichtmeier may I open a PR to add the flatpak stuff back into the repo? Because otherwise I'm almost certain this will run errors?

Reply:
sure

Reply:
I opened the PR.

Reply:
@hfiguiere if I replace the sha256 with the one from my commit,(d6f06d660fd5bf143425b79f5f8a93d328b34de3) would that work?

Reply:
Use this link

https://raw.githubusercontent.com/ubuntu-flutter-community/musicpod/d6f06d660fd5bf143425b79f5f8a93d328b34de3/flatpak/org.feichtmeier.Musicpod.appdata.xml